recipes/c4core/meta.yaml

```
@@ -0,0 +1,55 @@
+{% set name = "c4core" %}
+{% set version = "0.1.11" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://github.com/biojppm/c4core/archive/refs/tags/v{{ version }}.tar.gz
+    sha256: 65998b6ee62d20fe02a596dfd57d621d0387bdecf98ee1161d6e75858330500c
+
+  # NOT VENDORED! c4core relies on this repo by same developer as a git submodule. There are no releases, so we need to rely on git clone; depth 1 is enough.
+  - url: https://github.com/biojppm/cmake/archive/9059c98181f944aec83b90a677c96f3b43bd1047.zip
+    sha256: f265f0feb516729b513ef287abc52538cdef4963ab96edc7ce7bc56275851dde
+    folder: deps/cmake
+
+  # NOT VENDORED! c4core relies on this repo by same developer as a git submodule. There are no releases, so we need to rely on git clone; depth 1 is enough.
+  - url: https://github.com/biojppm/debugbreak/archive/5dcbe41d2bd4712c8014aa7e843723ad7b40fd74.zip
+    sha256: a9af29e2876c85103ada435aee6afd0698f0ad4282064a7c558a46f31bc86e25
+    folder: deps/debugbreak
+
+build:
+  number: 0
```

```suggestion
build:
  number: 0
  run_exports:
    - {{ pin_subpackage('c4core'), max_pin="x.x.x" }}
```
Build shared libraries not static libraries please, and adjust max_pin according to the ABI compatability guarantees of c4core.